ds000247/sub-0002/ses-0001/meg/sub-0002_ses-0001_coordsystem.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"MEGCoordinateSystem":"ctf","MEGCoordinateUnits":"cm","MEGCoordinateSystemDescription":"The origin is exactly between LPA and RPA; the X-axis goes towards NAS; the Y-axis goes approximately towards LPA, orthogonal to X and in the plane spanned by the fiducials; the Z-axis goes approximately towards the vertex, orthogonal to X and Y","EEGCoordinateSystem":"ctf","EEGCoordinateUnits":"cm","EEGCoordinateSystemDescription":"The origin is exactly between LPA and RPA; the X-axis goes towards NAS; the Y-axis goes approximately towards LPA, orthogonal to X and in the plane spanned by the fiducials; the Z-axis goes approximately towards the vertex, orthogonal to X and Y","IntendedFor":"/sub-0002/ses-0001/anat/sub-0002_ses-0001_T1w.nii.gz","HeadCoilCoordinates":{"coil1":[9.56534852,-0.02049955,0.02450495],"coil2":[-0.05239806,6.99865792,-0.02399622],"coil3":[0.08603075,-6.93959265,-0.0391808]},"HeadCoilCoordinateSystem":"ctf","HeadCoilCoordinateUnits":"cm","HeadCoilCoordinateSystemDescription":"The origin is exactly between LPA and RPA; the X-axis goes towards NAS; the Y-axis goes approximately towards LPA, orthogonal to X and in the plane spanned by the fiducials; the Z-axis goes approximately towards the vertex, orthogonal to X and Y","DigitizedHeadPoints":"/sub-0002/ses-0001/meg/sub-0002_ses-0001_task-rest_run-01_headshape.pos","DigitizedHeadPointsCoordinateSystem":"ctf","DigitizedHeadPointsCoordinateUnits":"cm","DigitizedHeadPointsCoordinateSystemDescription":"The origin is exactly between LPA and RPA; the X-axis goes towards NAS; the Y-axis goes approximately towards LPA, orthogonal to X and in the plane spanned by the fiducials; the Z-axis goes approximately towards the vertex, orthogonal to X and Y"} 
```

Comment:
seems like this should be upper case as well, also it'd greatly improve human readability to reformat this JSON to multiple lines, e.g., using https://jsonformatter.curiousconcept.com

```suggestion
{
  "MEGCoordinateSystem":"CTF",
  "MEGCoordinateUnits":"cm",
  "MEGCoordinateSystemDescription":"The origin is exactly between LPA and RPA; the X-axis goes towards NAS; the Y-axis goes approximately towards LPA, orthogonal to X and in the plane spanned by the fiducials; the Z-axis goes approximately towards the vertex, orthogonal to X and Y",
  "EEGCoordinateSystem":"CTF",
  "EEGCoordinateUnits":"cm",
  "EEGCoordinateSystemDescription":"The origin is exactly between LPA and RPA; the X-axis goes towards NAS; the Y-axis goes approximately towards LPA, orthogonal to X and in the plane spanned by the fiducials; the Z-axis goes approximately towards the vertex, orthogonal to X and Y",
  "IntendedFor":"/sub-0002/ses-0001/anat/sub-0002_ses-0001_T1w.nii.gz",
  "HeadCoilCoordinates":{
    "coil1":[
      9.56534852,
      -0.02049955,
      0.02450495
    ],
    "coil2":[
      -0.05239806,
      6.99865792,
      -0.02399622
    ],
    "coil3":[
      0.08603075,
      -6.93959265,
      -0.0391808
    ]
  },
  "HeadCoilCoordinateSystem":"CTF",
  "HeadCoilCoordinateUnits":"cm",
  "HeadCoilCoordinateSystemDescription":"The origin is exactly between LPA and RPA; the X-axis goes towards NAS; the Y-axis goes approximately towards LPA, orthogonal to X and in the plane spanned by the fiducials; the Z-axis goes approximately towards the vertex, orthogonal to X and Y",
  "DigitizedHeadPoints":"/sub-0002/ses-0001/meg/sub-0002_ses-0001_task-rest_run-01_headshape.pos",
  "DigitizedHeadPointsCoordinateSystem":"CTF",
  "DigitizedHeadPointsCoordinateUnits":"cm",
  "DigitizedHeadPointsCoordinateSystemDescription":"The origin is exactly between LPA and RPA; the X-axis goes towards NAS; the Y-axis goes approximately towards LPA, orthogonal to X and in the plane spanned by the fiducials; the Z-axis goes approximately towards the vertex, orthogonal to X and Y"
}
```

Reply:
Indeed, but my aim was to submit on PR per dataset, so that they can be rejected or approved separately: 
https://github.com/bids-standard/bids-examples/pull/218

(but then I committed-reverted things improperly to the wrong branches... sorry for the mess...)

Reply:
ahhh now I see it. No worries and thanks for the fixes. I'll keep an eye open and merge once the discussion has proceeded.

Reply:
Thanks @sappelhoff and @ftadel !!